config.nims

```diff
@@ -41,8 +56,9 @@ else:
   if defined(windows):
     # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65782
     # ("-fno-asynchronous-unwind-tables" breaks Nim's exception raising, sometimes)
-    switch("passC", "-mno-avx512vl")
-    switch("passL", "-mno-avx512vl")
+    switch("passC", "-mno-avx512f")
```

Comment:
Do we have a way to test this works?

Reply:
We are no building for Windows right now, so no, we do not really have a reasonable way to fix this. On the other hand this is "just" to prevent the compiler to enable these abx512f instructions which was the initial reason to look into this. Once we start building for Windows, we'll also want to verify the build artifacts work:) For now it is just a "dumb" copy of best practices from Nimbus:)